lib/exercism/user_finished_tracks.rb

```diff
@@ -24,9 +24,19 @@ def started_tracks
     ActiveRecord::Base.connection.execute(completed_count_sql).to_a
   end
 
+  def count_finished_problems(user_finished_exercises, track)
+    (user_finished_exercises & track.problems.map(&:slug)).size
+  end
+
+  def completed?(track_attributes)
```

Comment:
It's great! Since we are not going to call these methods outside our class it is a good idea to make them private. 👍